### ISSUE TITLE
Additional EKS nodes security nodes not attached to Ocean nodes

### ISSUE THREAD

**Issue by gastoncan** [spotinst/terraform-spotinst-ocean-eks#14]
```
module "ocean-eks" {
  source = "spotinst/ocean-eks/spotinst"
  worker_additional_security_group_ids = ["sg-xxxxxxxxxxx"]
 #redacted
}
```
When describing my EKS nodes, I don't get my additional SG sg-xxxxxxxxxxx.

Taking a look to the module code, I see the following (ocean.tf):
`security_groups             = [module.eks.worker_security_group_id]`

Maybe it should be, something like:
`security_groups  = concat([module.eks.worker_security_group_id], var.worker_additional_security_group_ids)`

A great job was done on this TF module, congrats!

**Comment by liranp** [comment]
Thank you for reporting this, @gastoncan. I've [fixed](https://github.com/spotinst/terraform-spotinst-ocean-eks/commit/d6d2d01c40b1f11f74a79a9e3a5a4ba03ba7a5a9) it and released version [1.3.0](https://registry.terraform.io/modules/spotinst/ocean-eks/spotinst/1.3.0).

**Comment by gastoncan** [comment]
Thanks for the quick fix! Appreciate that.
